tools/compress.py

```diff
@@ -167,17 +168,20 @@ def compress_rom(
         assert i <= len(compressed_rom_data)
         compressed_rom_data[segment_rom_start:i] = segment.data
 
+        rom_offset = segment_rom_end
+
+        if segment.is_syms:
+            segment_rom_start = 0xFFFFFFFF
+            segment_rom_end = 0xFFFFFFFF
+        elif not segment.is_compressed:
+            segment_rom_end = 0
+
         compressed_rom_dma_entries.append(
             dmadata.DmaEntry(
-                segment.vrom_start,
-                segment.vrom_end,
-                segment_rom_start,
-                segment_rom_end if segment.is_compressed else 0,
+                segment.vrom_start, segment.vrom_end, segment_rom_start, segment_rom_end
```

Comment:
```suggestion
                segment.vrom_start, segment.vrom_end, segment_rom_start, segment_rom_end,
```
And black-format so they're on single lines
The rationale behind one on each line is exactly to minimize diff noise